lib/article_class_fetcher.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+#= Fetches article classes from PageAssessments response
+# We simplify this parser by folding the nonstandard ratings
+# into the corresponding standard ones. We don't want to deal with edge cases
+# like bplus and a/ga.
+
+class ArticleClassFetcher
+	def initialize(pageassessments)
+		@assessments = pageassessments
```

Comment:
By changing this to `pageassessements || []` we can smoothly handle `nil` input.